CMakeLists.txt

```diff
@@ -1,177 +1,225 @@
-
-set(VITA_ELF_NAME ${SHORT_NAME}.elf)
-add_executable(${VITA_ELF_NAME}
-  ${PROJECT_RESOURCES}
-  ${PROJECT_SOURCE_FILES}
-)
-
-if(CMAKE_BUILD_TYPE MATCHES Debug)
-  message("Debug mode")
-  add_definitions(-DDEBUG)
-  if(DEBUGNET)
-    message("Debugnet turned on")
-    add_definitions(-DDEBUGNET)
-    target_link_libraries(${VITA_ELF_NAME} debugnet)
-    file(STRINGS "debugnetip.txt" DEBUGNETIP)
-    add_definitions(-DDEBUGNETIP="${DEBUGNETIP}")
-  endif()
-endif(CMAKE_BUILD_TYPE MATCHES Debug)
-
-target_link_libraries(${VITA_ELF_NAME}
-  yaml-cpp
-  m
-  vita2d
-  SceDisplay_stub
-  SceGxm_stub
-  SceSysmodule_stub
-  SceCtrl_stub
-  SceTouch_stub
-  ScePgf_stub
-  SceCommonDialog_stub
-  freetype
-  png
-  jpeg
-  z
-  m
-  c
-  SceNet_stub
-  SceNetCtl_stub
-  SceHttp_stub
-  SceSsl_stub
-  ${CURLPP_LDFLAGS}
-  ${CURL_LIBRARIES}
-  ${OPENSSL_LIBRARIES}
-  ftpvita
-  SceAppMgr_stub
-  SceAppUtil_stub
-  ScePromoterUtil_stub
-  SceIme_stub
-  ScePower_stub
-  SceAudio_stub
-  SceAudiodec_stub
-  SceVshBridge_stub
-  pthread
-)
-
-vita_create_self(${SHORT_NAME}.self ${VITA_ELF_NAME} UNSAFE UNCOMPRESSED)
-vita_create_vpk(${SHORT_NAME}.vpk ${VITA_TITLEID} ${SHORT_NAME}.self
-  VERSION ${VITA_VERSION}
-  NAME ${VITA_APP_NAME}
-  FILE sce_sys/icon0.png sce_sys/icon0.png
-  FILE sce_sys/livearea/contents/bg.png sce_sys/livearea/contents/bg.png
-  FILE sce_sys/livearea/contents/startup.png sce_sys/livearea/contents/startup.png
-  FILE sce_sys/livearea/contents/template.xml sce_sys/livearea/contents/template.xml
-
-  FILE assets/icons.zip resources/icons.zip
-  FILE assets/fonts/segoeui.ttf resources/fonts/segoeui.ttf
-)
-
-get_filename_component(psvitaipFilePath psvitaip.txt REALPATH)
-if(EXISTS "${psvitaipFilePath}")
-  file(STRINGS "${psvitaipFilePath}" PSVITAIP)
-
-  add_custom_target(send
-    COMMAND curl -T ${SHORT_NAME}.self ftp://${PSVITAIP}:1337/ux0:/app/${VITA_TITLEID}/eboot.bin
-    DEPENDS ${SHORT_NAME}.self
-  )
-
-  add_custom_target(shellsend
-    COMMAND psp2shell_cli ${PSVITAIP} 3333 load ${VITA_TITLEID} ${SHORT_NAME}.self
-    DEPENDS ${SHORT_NAME}.self
-  )
-
-  add_custom_target(vpksend
-    COMMAND curl -T ${SHORT_NAME}.vpk ftp://${PSVITAIP}:1337/ux0:/
-    DEPENDS ${SHORT_NAME}.vpk
-  )
-else()
-  message("Couldn't find psvitaip.txt. Put the IP-address of you vita there to get new build targets.")
-endif()
+cmake_minimum_required(VERSION 2.8)
+
+if(NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+  if(DEFINED ENV{VITASDK})
+    set(CMAKE_TOOLCHAIN_FILE "$ENV{VITASDK}/share/vita.toolchain.cmake" CACHE PATH "toolchain file")
+  else()
+    message(FATAL_ERROR "Please define VITASDK to point to your SDK path!")
+  endif()
+endif()
+
+
+set(SHORT_NAME VitaHBBrowser)
+project(${SHORT_NAME})
+include("${VITASDK}/share/vita.cmake" REQUIRED)
+
+set(ENV{PKG_CONFIG_PATH} "$ENV{VITASDK}/arm-vita-eabi/lib/pkgconfig")
+include(FindPkgConfig)
+
+set(VITA_APP_NAME "Vita Homebrew Browser")
+set(VITA_TITLEID "VHBB00001")
+set(UPDATER_TITLEID "VHBB00002")

```

Comment:
This title id is already used by a [fork](https://github.com/DevDavisNunez/vhbb/blob/d188411f66479611a2a018fbfac5163c04b4f441/CMakeLists.txt#L20).
I suggest you to use something like `VHBBUPDT1` to be safe.